### DISCUSSION TITLE
Is there a way to selectively disable some warnings?

### DISCUSSION THREAD

**Discussion by rsivan** [redhat-developer/vscode-java#2120] [Q&A]
For example my deprecated warnings blind my eyes and make it difficult to spot other warnings, so I would love to temporarily disable all deprecated warnings.

**Comment by 0dinD** [comment]
There's no way to do it easily via VS Code settings [yet](https://github.com/redhat-developer/vscode-java/issues/1791), but you could use an [Eclipse Preferences file](https://github.com/redhat-developer/vscode-java/wiki/Settings-Global-Preferences). Try adding `org.eclipse.jdt.core.compiler.problem.deprecation=ignore` to your `.prefs` file.

**Comment by Elziy** [comment]
You can suppress deprecation warnings by adding @SuppressWarnings("deprecation") to the class
